logger/logger.go

```diff
@@ -0,0 +1,63 @@
+package logger
```

Comment:
Recommendation: I assume it's better to use existing logging libraries like [logrus](https://github.com/sirupsen/logrus) if needed